tools/Deployment/deploy.js

```diff
@@ -432,12 +436,11 @@ function updateChocoConfigPromiseFn() {
 
 function pushChocoPackage() {
   return function() {
-    return new Promise(function(resolve, reject) {
+    return new Promise(function(reslove, reject) {
       if (!globalOptions.pkgName) {
         reject(new Error('package name can not be null/empty/undefined while pushing choco package'));
       }
-      let promiseFn = execPromiseFn('choco', ['push', globalOptions.pkgName], config.choco.homeDir);
-      promiseFn().then(resolve).catch(reject);
+      return execPromiseFn('choco', ['push', globalOptions.pkgName], config.choco.homeDir)();
```

Comment:
Looks like something from the auto rebase tooling. I will look into it manually.